nx-X11/programs/Xserver/hw/nxagent/Keyboard.c

```diff
@@ -1863,10 +1863,9 @@ void nxagentKeycodeConversionSetup(void)
     fprintf(stderr, "Failed to create the keyboard file\n");
   }
 
-  if (nxagentOption(ClientOs) == ClientOsLinux &&
-            drules != NULL && dmodel != NULL &&
-                (strcmp(drules, "evdev") == 0 ||
-                    strcmp(dmodel, "evdev") == 0))
+  if (drules != NULL && dmodel != NULL &&
```

Comment:
Mh, careful, at least when it comes to the commit message. `evdev` will also be supported on FreeBSD once a new version is released, so we can't classify it as "Linux-only". This said, there's no `clientOs` value for *BSD currently, so I assume it will default to `UNDEFINED (-1)` on that platform.